### ISSUE TITLE
kedro docker build

### ISSUE THREAD

**Issue by annemaryjose** [quantumblacklabs/kedro#854]
## Description
Trying to do a kedro docker build for the spaceflight-tutorial. I run into the error.

#8 37.00   Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
#8 37.17 Collecting python-json-logger==2.0.2
#8 37.21   Downloading python_json_logger-2.0.2-py3-none-any.whl (7.4 kB)
#8 37.32 Collecting python-slugify==5.0.2
#8 37.37   Downloading python_slugify-5.0.2-py2.py3-none-any.whl (6.7 kB)
#8 37.70 Collecting pytz==2021.1
#8 37.73   Downloading pytz-2021.1-py2.py3-none-any.whl (510 kB)
#8 37.96 ERROR: Could not find a version that satisfies the requirement pywin32==301 (from versions: none)
#8 37.96 ERROR: No matching distribution found for pywin32==301
 **ERROR: Could not find a version that satisfies the requirement pywin32==301**

## Context
Unable to build image !

## Steps to Reproduce
1.execute kedro docker  build at root of spaceflight tutorial

## Expected Result
Image should be built

## Actual Result
Mentioned above

```
-- If you received an error, place it here.
```

```
-- Separate them if you have more than one.
```

## Your Environment
Include as many relevant details about the environment in which you experienced the bug:

* Kedro version used (`pip show kedro` or `kedro -V`):
* Python version used (`python -V`):
* Operating system and version:

**Comment by annemaryjose** [comment]
python 3.7.11, kedro 0.17.4 , OS Windows 10 - 19042.
Docker - latest available on the internet for windows v 20.10.8

**Comment by annemaryjose** [comment]
adding  sys_platform == "win32" and python_version >= "3.6" or sys_platform == "win32" to the dependency made it work.
